### DISCUSSION TITLE
Create Custom Module

### DISCUSSION THREAD

**Discussion by Abedkingfc91** [synthetichealth/synthea#1364] [Q&A]
How do I create a custom module to generate 50 patients with cystic fibrosis in Austin texas and include the following resources  - Patient, Condition, Observation, Medication, FamilyMemberHistory, DiagnositcReport, Procedure, Practitioner, CarePlan, AllergyIntolerance

**Comment by jawalonoski** [comment]
There already is a cystic fibrosis module: https://synthetichealth.github.io/module-builder/#cystic_fibrosis

You can use that as-is, or you can customize it until your heart is content.

FHIR is output by default, except there is no support for FamilyMemberHistory.

You probably want to define a keep module that specifies cystic fibrosis (with the SNOMED code `190905008` and description `Cystic Fibrosis`). Name it something like `keep_cf.json`.

See keep module instructions: https://github.com/synthetichealth/synthea/wiki/Keep-Patients-Module

Then, once all that is in place:

```
./run_synthea -p 50 -k keep_cf.json Texas Austin
```
